sdk/src/main/java/com/qonversion/android/sdk/storage/DefinedUserProperties.kt

```diff
@@ -0,0 +1,5 @@
+package com.qonversion.android.sdk.storage
+
+enum class DefinedUserProperties(val userPropertyCode: String) {
+    EMAIL("_q_email"), NAME("_q_name");
```

Comment:
Let's add all defined user properties. We need:

"_q_name";
"_q_kochava_device_id"
"_q_appsflyer_user_id"
"_q_adjust_adid"
"_q_custom_user_id"